config/config.ts

```diff
@@ -117,7 +117,7 @@ const config: Configuration = {
                                 'https://github.com/flybywiresim/a32nx/releases/download/assets/experimental/',
                             ],
                             description: 'Development will have the latest features that will end up in the next stable. ' +
-                                'Bugs are to be expected. It updates whenever something is added to the \'master\' ' +
+                                'Although every change is QA-tested bugs are a little more likely. It updates whenever something is added to the \'master\' ' +
```

Comment:
```suggestion
                                'Although every change is QA-tested, bugs are a little more likely. It updates whenever something is added to the \'master\' ' +
```
Add comma for better reading flow